src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -411,15 +430,14 @@ public void populateCaseWorkerProfile(CaseWorkersProfileCreationRequest cwrdProf
         return caseWorkerWorkAreas;
     }
 
-    public boolean updateUserProfile(CaseWorkersProfileCreationRequest cwrdProfileRequest,
+    public CaseWorkerProfile updateUserProfile(CaseWorkersProfileCreationRequest cwrdProfileRequest,
                                      CaseWorkerProfile caseWorkerProfile) {
```

Comment:
Refactor method name it is very misleading and hard to maintain

Reply:
yes , anyways this code will be refactored in review comments change.